httpbakery/export_test.go

```diff
@@ -5,3 +5,5 @@ type PublicKeyResponse publicKeyResponse
 const MaxDischargeRetries = maxDischargeRetries
 
 var LegacyGetInteractionMethods = legacyGetInteractionMethods
+
+var SeekerFromBody = seekerFromBody
```

Comment:
I can see why this is attractive, but is it difficult to use a real `http.Request` (maybe [`httptest.NewRequest`](https://pkg.go.dev/net/http/httptest#NewRequest)) so this test is more real, and to avoid testing internals?

Reply:
Done